recipe_scrapers/_schemaorg.py

```diff
@@ -77,6 +83,18 @@ def get_key_and_minutes(k):
             total_time = sum(times)
         return total_time
 
+    def cook_time(self):
+        if not (self.data.keys() & {"cookTime"}):
```

Comment:
This took a few moments to understand - could `if "cooktime" not in self.data.keys()` be a simpler way to express this?

Reply:
Hi Jay, did you check the original code for function total_time in the same file? I copied the same code as used there. I also think this is hard to read.

Reply:
@jayaddison is there something I have to do now? Regards.